packages/@divvi/mobile/src/navigator/NavigationService.ts

```diff
@@ -234,40 +234,11 @@ export async function isBottomSheetVisible(screen: Screens) {
 export function navigateHome(fromModal?: boolean) {
   const timeout = fromModal && Platform.OS === 'ios' ? 500 : 0
   setTimeout(() => {
-    navigateClearingStack(Screens.TabNavigator, { initialScreen: Screens.TabHome })
+    navigateClearingStack(Screens.TabNavigator)
```

Comment:
Wonder if we should have a different helper for navigating to `TabHome` (which is the activity tab) and use that for tx submission.

Reply:
I was thinking we should rename `TabHome` -> `TabActivity` .
And `navigateHome` -> `navigateInitialTab`.
What do you think?

Reply:
sure, I guess screen name change is backwards compatible with analytics with the screen events, but I guess we generally don't use that for metrics

Reply:
ah right yes,
would be good to rename if we don't really rely on it for metrics.

Reply:
chatted with John and it looks like they do use screen events, I am going to hold off on the screen rename for now.

Reply:
yes would be nice to know. But we can always do that later.